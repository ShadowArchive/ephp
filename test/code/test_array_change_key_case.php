<?php
$input_array = array("FirSt" => 1, "SecOnd" => 4);
print_r(array_change_key_case($input_array, CASE_UPPER));

$input_array = array("one", "two");
print_r(array_change_key_case($input_array, CASE_LOWER));

$input_array = array("fIrSt" => 10, "sEcOnD" => 20);
print_r(array_change_key_case($input_array));
print_r(array_change_key_case($input_array, 100));
var_dump(array_change_key_case("hello!"));

