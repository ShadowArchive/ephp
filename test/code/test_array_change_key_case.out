Array
(
    [FIRST] => 1
    [SECOND] => 4
)
Array
(
    [0] => one
    [1] => two
)
Array
(
    [first] => 10
    [second] => 20
)
Array
(
    [FIRST] => 10
    [SECOND] => 20
)

Warning: array_change_key_case() expects parameter 1 to be array, string given in {{CWD}}/test/code/test_array_change_key_case.php on line 11
NULL
